### ISSUE TITLE
Data query on ec-devco spend

### ISSUE THREAD

**Issue by bill-anderson** [IATI/IATI-Dashboard#380]
http://dev.dashboard.iatistandard.org/transparencyindicator.html

European Commission - Development and Cooperation-EuropeAid (ec-devco) shows no IATI spend for  2014 or 15

Random check showed up Activity - EU-1-2014/350-097 in  Afghanistan with this transaction

Transaction: [ref="EU-1-2014/350-097/DCI-ENV/2014/031-104" ]
Transaction-date: [iso-date="2014-11-14" ] 2014-11-14
Transaction-type: [code="3" ] Disbursements
Value: [value-date="2014-11-14" ] 500000
Receiver-org:  United Nations Office For Project Services

**Comment by dalepotter** [comment]
Thanks for spotting this one Bill. I've done some investigation on the transaction you highlighted from your spot check.  The activity (identifier `EU-1-2014/350-097`) is contained within the [ec-devco-298 dataset](http://iatiregistry.org/dataset/ec-devco-298). 

This issue arises as, whilst the publisher have specified their data is published at `version="1.04"` (in both the `<iati-activity>` and their `<iati-activities>` elements that form part of this dataset), they are using version 2.x codelist values for the transaction-type/@code attribute. The transaction therefore fails [this part](https://github.com/IATI/IATI-Stats/blob/master/stats/dashboard.py#L931) of the logic test which determines if the transaction should be counted.

Perhaps @iatisupport or @rory09 would be able to highlight this issue to ec-devco. The current situation means that this publisher will not gain any credit for the value of the data that they are reporting to IATI.

![image](https://cloud.githubusercontent.com/assets/8247168/13003917/24e26b8c-d170-11e5-949c-7f31db39155a.png)

An extract of the raw data for activity and the aforementioned transaction follows:

```
<iati-activity default-currency="EUR" last-updated-datetime="2016-01-08T11:31:23" version="1.04" xml:lang="en">
    <reporting-org ref="XI-IATI-EC_DEVCO" type="15">Commission of the European Communities</reporting-org>
    <iati-identifier>EU-1-2014/350-097</iati-identifier>

    [...]

    <transaction ref="EU-1-2014/350-097/DCI-ENV/2014/031-104">
        <transaction-date iso-date="2014-11-14">2014-11-14</transaction-date>
        <transaction-type code="3">Disbursements</transaction-type>
        <value value-date="2014-11-14">500000</value>
        <receiver-org>United Nations Office For Project Services</receiver-org>
    </transaction>

    [...]

</iati-activity>
```

With this in mind, so I will close this issue as this is not classed as a bug. Do let me know if there is more data to investigate.

**Comment by iatisupport** [comment]
Just to add that the data error appears to have been fixed now. See http://dashboard.iatistandard.org/publisher/ec-devco.html as no transaction type errors reported and also values now reported for coverage annual spend figures
